src/GafferScene/Instancer.cpp

```diff
@@ -2736,7 +2731,15 @@ void Instancer::InstancerCapsule::render( IECoreScenePreview::Renderer *renderer
 				!hasAttributes
 			);
 		},
-		limits<size_t>::max() // Never evict, even if prototypes are all unique
+
+		std::numeric_limits<size_t>::max(), // Never evict, even if prototypes are all unique
```

Comment:
Yeah, I'm not even sure what `limits` was finding, other than that it must have been a large integer, since the tests passed. I only fixed here because it was a last minute fix to get CI going, and I was planning to squash anyway. I've now fixed it in the correct place.